packages/eslint-config-next/index.js

```diff
@@ -95,11 +96,8 @@ module.exports = {
       files: ['**/*.ts?(x)'],
       parser: '@typescript-eslint/parser',
       parserOptions: {
+        project: true,
```

Comment:
Does this do additional work even if type-checked rules are disabled? If yes, is that blocking?

Reply:
Oop, you're right, this is vestigial.